modules/ob-creating-a-s2i-build.adoc

```diff
@@ -43,6 +43,12 @@ spec:
     pushSecret: registry-credential <5>
 EOF
 ----
+<1> The location where the source code is placed.
+<2> The build strategy that you use to build the container.
+<3> The parameter defined in the build strategy. To set the value of the `builder-image` strategy parameter, specify the builder image location required to build the output image.
+<4> The location where the built image is pushed. You can push the built image to a custom Quay repo. Replace `repo` with a valid Quay organization or your Quay user name.
+<5> The secret name that stores the credentials for pushing container images. To generate a secret of the type `docker-registry` for authentication, see _Authentication to container registries_ in the _Additional resources_ section.
```

Comment:
In the past I've been told we don't need to worry about saying "In the additional resources section." Also, OCP article titles typically go in double quotations. 

```suggestion
<5> The secret name that stores the credentials for pushing container images. To generate a secret of the type `docker-registry` for authentication, see "Authentication to container registries".
```